R/dropdown.R

```diff
@@ -41,27 +41,25 @@ dropdown_input <- function(input_id, choices, choices_value = choices,
       icon("dropdown"),
       shiny::div(class = "default text", default_text),
       menu(
-        purrr::when(
-          choices,
-          is.null(names(.)) ~
-            purrr::map2(
-              choices, choices_value,
-              ~ shiny::div(class = "item", `data-value` = .y, .x)
-            ),
-          !is.null(names(.)) ~
-            purrr::map(
-              seq_len(length(choices)), ~ {
-                shiny::tagList(
-                  menu_header(names(choices)[.x], is_item = FALSE),
-                  menu_divider(),
-                  purrr::map2(
-                    choices[[.x]], choices_value[[.x]],
-                    ~ shiny::div(class = "item", `data-value` = .y, .x)
-                  )
+        if (is.null(names(choices))) {
```

Comment:
Style comment:
There are places in which we are using explicitly namespaced functions like here `purrr::map` and other places in which we don't. Why not using `@importFrom purrr map map2 etc...` in here?
If there is a reason to do so then it's fine, if not, then maybe we should start to consolidate the style.
If you think we should consolidate it then I can create an issue for it